### PR TITLE
QPAY-179: Added support to create an account with a custom name

### PR DESCRIPTION
**Pull request by WJRobert**
Added support to create an account with a custom name

### CODE REVIEW

File: Nexus.Sdk.Token/TokenServerProvider.cs [hunk 1]
```diff
@@ -6,6 +6,7 @@
 using Nexus.Sdk.Shared.Responses;
 using Nexus.Sdk.Token.Requests;
 using Nexus.Sdk.Token.Responses;
+using System.Security.AccessControl;
```
Comment (nagarwal4) [inline Nexus.Sdk.Token/TokenServerProvider.cs:9]: This namespace?

**Review by nagarwal4 — CHANGES_REQUESTED** [review]

File: Nexus.Sdk.Token/TokenServerProvider.cs (hunk 1 above)
Comment (WJRobert) [reply Nexus.Sdk.Token/TokenServerProvider.cs]: Thank you for spotting that one, I should have picked it up before sending to review.

I removed it :)

File: Nexus.Sdk.Token.Tests/Helpers/MockTokenServerProvider.cs [hunk 2]
```diff
@@ -22,12 +22,12 @@ public Task<SignableResponse> ConnectAccountToTokensAsync(string accountCode, IE
...
         }
 
-        public Task<AccountResponse> CreateAccountOnAlgorandAsync(string customerCode, string publicKey, string? customerIPAddress = null)
+        public Task<AccountResponse> CreateAccountOnAlgorandAsync(string customerCode, string publicKey, string? customerIPAddress = null, string? customName = null)
         {
             throw new NotImplementedException();
         }
 
-        public Task<SignableResponse> CreateAccountOnAlgorandAsync(string customerCode, string publicKey, IEnumerable<string> tokenCodes, string? customerIPAddress = null)
+        public Task<SignableResponse> CreateAccountOnAlgorandAsync(string customerCode, string publicKey, IEnumerable<string> tokenCodes, string? customerIPAddress = null, string? customName = null)
```
Comment (nagarwal4) [inline Nexus.Sdk.Token.Tests/Helpers/MockTokenServerProvider.cs:30]: Should we also add it to the Stellar ones?

Comment (WJRobert) [reply Nexus.Sdk.Token.Tests/Helpers/MockTokenServerProvider.cs:30]: Good idea, I have added the stellar ones :)

File: Nexus.Sdk.Token/Facades/AccountsFacade.cs [hunk 3]
```diff
@@ -40,14 +40,14 @@ public async Task<SignableResponse> CreateOnStellarAsync(string customerCode, st
         return await _provider.CreateAccountOnStellarAsync(customerCode, publicKey, allowedTokens, customerIPAddress);
     }
 
-    public async Task<AccountResponse> CreateOnAlgorandAsync(string customerCode, string publicKey, string? customerIPAddress = null)
+    public async Task<AccountResponse> CreateOnAlgorandAsync(string customerCode, string publicKey, string? customerIPAddress = null, string? customName = null)
```
Comment (nagarwal4) [inline Nexus.Sdk.Token/Facades/AccountsFacade.cs:43]: Commenting here: Stellar ones are missing customeName property

File: Nexus.Sdk.Token/Facades/Interfaces/IAccountsFacade.cs [hunk 4]
```diff
@@ -48,8 +48,9 @@ public interface IAccountsFacade
     /// <param name="customerCode">Unique Nexus identifier of the customer.</param>
     /// <param name="publicKey">The public key the new Algorand account</param>
     /// <param name="customerIPAddress">Optional IP address of the customer used for tracing their actions</param>
+    /// <param name="customName">Optional custom name for account</param>
     /// <returns>The Nexus account that is created</returns>
-    public Task<AccountResponse> CreateOnAlgorandAsync(string customerCode, string publicKey, string? customerIPAddress = null);
+    public Task<AccountResponse> CreateOnAlgorandAsync(string customerCode, string publicKey, string? customerIPAddress = null, string? customName = null);
```
Comment (nagarwal4) [inline Nexus.Sdk.Token/Facades/Interfaces/IAccountsFacade.cs:53]: Also in this file, Stellar ones are missing customName property

**Review by nagarwal4 — APPROVED** [review]

**Pull request merged**